### ISSUE TITLE
Fort Avenue sidewalk crossing of Hutters Lane is dangerous, confusing, and nearly intraversible in a wheelchair

### ISSUE THREAD

**Issue by partytax** [partytax/lynchburg-road-issues#42]
https://www.openstreetmap.org/#map=19/37.40417/-79.15785

The concrete surface of Hutters Lane, as well as its joint with the asphalt surface of Fort Avenue, are extremely degraded with multiple cracks and potholes. This makes crossing Hutters Lane fraught for wheelchair users and pedestrians.

Additionally, on the northeast side of the crossing, the lowered ADA curb slants toward moving traffic on Fort Avenue rather than into a crossing of Hutters Lane.

Finally, the ADA tactile pad on the northeast side of the crossing is so far from the actual crossing that it would be unclear to a blind person where the crossing actually was. It's almost far enough northeast to be a tactile pad for the crossing of the Slayton's Automotive driveway, but I don't think it was installed for that because there's no tactile pad on the other side of the driveway.

![signal-2023-05-03-211105_002](https://github.com/partytax/lynchburg-road-issues/assets/29897671/81bc8b45-951e-4215-9f98-58d4013f65da)
![signal-2023-05-03-211043_002](https://github.com/partytax/lynchburg-road-issues/assets/29897671/aae59daf-2136-43da-9b03-4503dceeb3c3)
![signal-2023-05-03-211031_002](https://github.com/partytax/lynchburg-road-issues/assets/29897671/6200cc68-39c1-4ac9-a4dd-48197b2fa2c7)

**Comment by partytax** [comment]
Emailed Lynchburg Citizens First Information Center:

```
ATTN Citizens First Information Center:
Please forward the section below to the Public Works Department, providing a confirmation/issue number if possible. Thank you.

ATTN Public Works:
On Fort Avenue, at the T-section with Hutters Lane, the pedestrian crossing of Hutters Lane is in extremely poor condition. This is mostly due to the broken, concrete pavement on Hutters Lane. Please repair the surface of the crossing, and if possible align the curb ramps so they direct wheelchair users in a straight line across the lane and not into Fort Avenue traffic. For pictures and more comments, please go to

lri [dot] lynchburg [dot] xyz
Issue #42

Thank you for your time.
```

**Comment by partytax** [comment]
Spoke to Tim Yarn yesterday about another issue and brought up this issue as well, requesting that he take a look at it.

**Comment by partytax** [comment]
Has been about 1.5 months since last reminding public works department of this issue. Will wait a couple of months before requesting status.

**Comment by partytax** [comment]
Considered that email may never have been received by Citizens First, since they never responded. Emailed them again:

```
I submitted this issue on August 12, 2023, but never received a response. I'm guessing the email was blocked by your firewall.

On Fort Avenue, at the T-section with Hutters Lane, the pedestrian crossing of Hutters Lane is in extremely poor condition. This is mostly due to the broken, concrete pavement on Hutters Lane. Please repair the surface of the crossing, and if possible align the curb ramps so they direct wheelchair users in a straight line across the lane and not into Fort Avenue traffic. For reference this is Lynchburg Road Issue #42.

Thank you for your time.
```

**Comment by partytax** [comment]
Received response from Citizens First;

```
Good Morning,

 

This has been reported to Public Works. Work Request #23-012422.

 

Thank you,

 

Chonna N. Kidd

Senior Customer Service Representative

From: Nathan Wyand <me@nathanwyand.com>
Sent: Wednesday, December 20, 2023 11:03 AM
To: Citizens First <citizensfirst@lynchburgva.gov>
Subject: ATTN: Public Works

 

CAUTION: External Sender

 

I submitted this issue on August 12, 2023, but never received a response. I'm guessing the email was blocked by your firewall.

On Fort Avenue, at the T-section with Hutters Lane, the pedestrian crossing of Hutters Lane is in extremely poor condition. This is mostly due to the broken, concrete pavement on Hutters Lane. Please repair the surface of the crossing, and if possible align the curb ramps so they direct wheelchair users in a straight line across the lane and not into Fort Avenue traffic. For reference this is Lynchburg Road Issue #42.

Thank you for your time.
```

**Comment by partytax** [comment]
Requested update with Citizens First by email:

>Good evening. Could I have a status update on work request #23-012422 ? The road is still in very poor condition at the location of the pedestrian crossing. Thank you for your time.

**Comment by partytax** [comment]
Received response from Citizens First:

>Good Morning,
I spoke with Timyarn Benjamin & he says it is on the list to be repaired. No time frame as of yet.
Thank you,
Chonna

**Comment by partytax** [comment]
Not fixed as of the other day. Will wait a while longer before checking in, as it is on "the list to be repaired".

**Comment by partytax** [comment]
Not fixed yet. Made task to call Tim Yarn Benjamin again to inquire about status.

**Comment by partytax** [comment]
Called and spoke to Timyarn Benjamin about this. He said it's on the list but they (probably Counts & Dobyns or other contractor) have a huge backlog of work to do.

**Comment by partytax** [comment]
Not fixed. FOIA-ing for "the list"

**Comment by partytax** [comment]
Sent request to foia@lynchburgva.gov:

>Provide the record or document containing the current list of sidewalk repair projects the city will complete or has contracted with an outside company to complete. If possible, include the location, date the repair was requested, and date the repair is scheduled to be completed. CSV or Excel format is preferred.

**Comment by partytax** [comment]
Received confirmation for FOIA request today

**Comment by partytax** [comment]
Received document, but it doesn't contain this project

**Comment by partytax** [comment]
Not going to follow up at this time because it's "on the list"
